recipe/meta.yaml

```diff
@@ -9,30 +9,33 @@ source:
   # we have to use git urls here to get the submodules needed for the
   # build
   git_url: https://github.com/dmlc/xgboost
-  git_tag: 1bf389998383f333490155dba4608bff9ca63b42
+  git_tag: v1.3.0
   patches:
     # xgboost patches
     - 0001-conda-Unbundle-libxgboost.-dll-dylib-so.patch
     - 0002-Fix-R-package-PKGROOT.patch
 
 build:
   number: 1
-  skip: true  # [win or linux32 or py2k]
+  skip: true  # [linux32 or py2k]
```

Comment:
You can completely remove this line now. The other two are skipped by default nowadays.
```suggestion
```